Formula 1/Views/Cells/CircuitCell.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  CircuitCell.swift
+//  Formula 1
+//
+//  Created by Gio on 11/13/19.
+//  Copyright © 2019 Gio. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CircuitCell: UICollectionViewCell {
+    static let reuseIdentifier = String(describing: self)
+    
+    lazy var nameLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.adjustsFontForContentSizeCategory = true
+        label.font = UIFont.Formula1Font.Regular
+        return label
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setupUI()
+        applySkin()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension CircuitCell {
+    func setupUI() {
+          contentView.addSubview(nameLabel)
+
+          NSLayoutConstraint.activate([
+              nameLabel.centerXAnchor.constraint(equalTo: contentView.centerXAnchor),
+              nameLabel.centerYAnchor.constraint(equalTo: contentView.centerYAnchor)
+          ])
+      }
+      
+      func applySkin() {
+          backgroundColor = .systemGray6
+          
+          layer.borderWidth = 0.2
```

Comment:
Subpixel initializing makes things blurry, stick to full points. You can just leave a TODO to fix later later

Reply:
added warning